app/main.py

```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
```

Comment:
It's better to wrap a long line into two parts than into 5, imho

Reply:
okey, understood, thanks